pkg/plugins/plugins_windows.go

```diff
@@ -22,7 +23,8 @@ func RegisterPlugins(a *agent.Agent) error {
 	}
 
 	// Enabling the hostinfo plugin will make the host appear in the UI
-	a.RegisterPlugin(pluginsWindows.NewHostinfoPlugin(ids.PluginID{"metadata", "system"}, a.Context, a.GetCloudHarvester()))
+	a.RegisterPlugin(pluginsWindows.NewHostinfoPlugin(ids.PluginID{"metadata", "system"}, a.Context,
```

Comment:
not critical, but I would call `NewHostinfoPlugin` in the same way as in linux/macos ☝️